pkg/interface/src/logic/state/local.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import create  from 'zustand';
+import { persist } from 'zustand/middleware';
+import produce from 'immer';
+import { BackgroundConfig, RemoteContentPolicy } from "~/types/local-update";
+
+export type LocalState = {
+  hideAvatars: boolean;
+  hideNicknames: boolean;
+  remoteContentPolicy: RemoteContentPolicy;
+  dark: boolean;
+  background: BackgroundConfig;
+  omniboxShown: boolean;
+  suspendedFocus?: HTMLElement;
```

Comment:
this should be an interface not a type

Reply:
If I do that I get this:
<img width="768" alt="Screen Shot 2020-12-17 at 9 50 45 AM" src="https://user-images.githubusercontent.com/3999320/102524108-5d7c0380-404d-11eb-8d77-efceaee395d0.png">

How do I fix this?

Reply:
https://github.com/pmndrs/zustand/issues/213